alacritty/src/renderer/builtin_font.rs

```diff
@@ -20,12 +20,20 @@ pub fn builtin_glyph(
     character: char,
     metrics: &Metrics,
     offset: &Delta<i8>,
+    glyph_offset: &Delta<i8>,
 ) -> Option<RasterizedGlyph> {
-    match character {
+    let mut glyph = match character {
         // Box drawing characters and block elements.
-        '\u{2500}'..='\u{259f}' => Some(box_drawing(character, metrics, offset)),
-        _ => None,
-    }
+        '\u{2500}'..='\u{259f}' => box_drawing(character, metrics, offset),
+        _ => return None,
+    };
+
+    // Since we want to ignore `glyph_offset` for builtin font apply it to compensate its addition
+    // when loading glyphs in the renderer.
+    glyph.left -= glyph_offset.x as i32;
+    glyph.top -= glyph_offset.y as i32;
```

Comment:
That's pretty hacky, but I guess it's the easiest way?

Reply:
Yeah, but that doesn't change glyph loading. Otherwise you have to pass this delta and whether it's builtin or not around, and gets very ugly pretty soon.